src/metrics.jl

```diff
@@ -802,7 +802,7 @@ end
 
 # CosineDist
 
-function _pairwise!(r::AbstractMatrix, dist::CosineDist,
+function _pairwise!(r::AbstractMatrix, ::CosineDist,
```

Comment:
Do we use the same style elsewhere?

Reply:
Yes, pretty much everywhere, like in the `eval_*` section of the same file, whenever we don't need `dist` explicitly, but only dispatch on the type in that argument slot.